flocker_bb/builders/flocker_vagrant.py

```diff
@@ -381,6 +447,11 @@ def slave_class(self):
             return 'aws/centos-7'
 
 
+CLIENT_INSTALLATION_CONFIGURATIONS = [
+    ClientConfiguration(
+        provider='rackspace', distribution='ubuntu-14.04'),
```

Comment:
Should we support and add a build slave for Ubuntu 14.10 too?
There seem to be installation problems on that platform:
 * https://clusterhq.atlassian.net/browse/FLOC-1989